README.md

```diff
@@ -30,7 +30,7 @@ This repository contains a basic Monitoring setup [Rocket.Chat](https://github.c
 4. Create and start up containers using `docker-compose`. Make sure to include *both* compose files:
 
     ```
-    docker-compose up -d
+    docker-compose -f docker-compose.yml -f docker-compose.monitoring.yml up -d
```

Comment:
I got rid of the chained compose files to make sure this works as an independent metric setup, no matter the actual Rockte.Chat deployment.

Reply:
Ok, that makes sense. Then the the phrasing 'Make sure to include *both* compose files:' should be changed in the README I guess.